Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -39,181 +42,210 @@
+          "MessageArguments": { "secretKind": "GitHub app id and secret" },
+          "HelpUri": "https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/token-expiration-and-revocation"
         },
         {
           "Id": "SEC101/008",
           "Name": "DoNotExposePlaintextSecrets/AwsCredentials",
           "IntrafileRegexes": [ "$SEC101/008.AwsCredentialsId", "$SEC101/008.AwsCredentialsSecret" ],
-          "MessageArguments": { "secretKind": "Aws access key and secret" }
+          "MessageArguments": { "secretKind": "Aws access key and secret" },
+          "HelpUri": "https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html"
         },
         {
           "Id": "SEC101/009",
           "Name": "DoNotExposePlaintextSecrets/LinkedInCredentials",
           "IntrafileRegexes": [ "$SEC101/009.LinkedInCredentialsId", "$SEC101/009.LinkedInCredentialsSecret" ],
-          "MessageArguments": { "secretKind": "LinkedIn access key and secret" }
+          "MessageArguments": { "secretKind": "LinkedIn access key and secret" },
+          "HelpUri": "https://developer.linkedin.com/support/faq"
         },
         {
           "Id": "SEC101/010",
           "Name": "DoNotExposePlaintextSecrets/SquarePat",
           "ContentsRegex": "$SEC101/010.SquarePat",
-          "MessageArguments": { "secretKind": "Square personal access token" }
+          "MessageArguments": { "secretKind": "Square personal access token" },
+          "HelpUri": "https://developer.squareup.com/docs/build-basics/access-tokens"
         },
         {
           "Id": "SEC101/011",
           "Name": "DoNotExposePlaintextSecrets/SquareCredentials",
           "IntrafileRegexes": [ "$SEC101/011.SquareCredentialsId", "$SEC101/011.SquareCredentialsSecret" ],
-          "MessageArguments": { "secretKind": "Square access key and secret" }
+          "MessageArguments": { "secretKind": "Square access key and secret" },
+          "HelpUri": "https://developer.squareup.com/docs/build-basics/access-tokens"
         },
         {
           "Id": "SEC101/012",
           "Name": "DoNotExposePlaintextSecrets/SlackWebhook",
           "ContentsRegex": "$SEC101/012.SlackWebhook",
-          "MessageArguments": { "secretKind": "Slack web hook" }
+          "MessageArguments": { "secretKind": "Slack web hook" },
+          "HelpUri": "https://slack.com/help/articles/215770388-Create-and-regenerate-API-tokens"
         },
         {
           "Id": "SEC101/013",
           "Name": "DoNotExposePlaintextSecrets/CryptographicPrivateKey",
           "ContentsRegex": "$SEC101/013/Pem.CryptographicPrivateKey",
           "MessageArguments": { "secretKind": "PEM encoded private key" },
-          "Properties": { "kind": "Default" }
+          "Properties": { "kind": "Default" },
+          "HelpUri": "https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key"
```

Comment:
>https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key [](http://example.com/codeflow?start=22&length=105)

i don't think this 013 links are correct.

Reply:
Done